plugin/pkg/admission/imagepolicy/admission.go

```diff
@@ -219,6 +219,10 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 // For additional HTTP configuration, refer to the kubeconfig documentation
 // http://kubernetes.io/v1.1/docs/user-guide/kubeconfig-file.html.
 func NewImagePolicyWebhook(configFile io.Reader) (*Plugin, error) {
+	if configFile == nil {
+		return nil, fmt.Errorf("No config specified")
```

Comment:
sorry, the case in my message https://github.com/kubernetes/kubernetes/pull/55379#discussion_r151248793 was important.  `fmt.Errorf` message always start lower case and end without punctuation.

Reply:
Oh, gotcha. I might have seen lots of wrong demonstration :)